custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -39,6 +41,14 @@ def __init__(self):
     def configure(self, params):
         super(JavaPredictor, self).configure(params)
 
+        ## retrieve the extension of the main java model artifact
+        self.custom_model_path = params["__custom_model_path__"]
+        ext_re = re.search(
```

Comment:
This code doesn't work with the codegen jar model.
Here are print's I added
```
ext_re = re.search(
            "({})|({})|({})".format(*JavaArtifacts.ALL),
            ",".join(os.listdir(self.custom_model_path)),
        )
        print(",".join(os.listdir(self.custom_model_path)))
        print(ext_re)
        self.model_artifact_extension = ext_re.group()
        print(self.model_artifact_extension)
        exit(1)
```
And this is output:
```
README.md,java_reg.jar
<_sre.SRE_Match object; span=(9, 14), match=',java'>
,java
```

Reply:
Update:
it passes in tests, because there model dir only contains one `java_reg.jar` file
So you could just test that code snippet on a dir with different number of files with different extensions.

Reply:
@yakov-g i tested this out locally on java-codegen which contained the jar and readme and works fine.  It is odd that it picked up `,java` since java artifacts being leverages in the regex are are 
`class JavaArtifacts(object):
    JAR_EXTENSION = ".jar"
    MOJO_EXTENSION = ".zip"
    POJO_EXTENSION = ".java"
    ALL = [JAR_EXTENSION, MOJO_EXTENSION, POJO_EXTENSION]`

Reply:
regex used to identify the extension has been fixed.  Forgot to the escape `.` in the expression.